docs/development/minikube.md

```diff
@@ -0,0 +1,90 @@
+# Running Development Environment in Kubernetes
+
+## Start Minikube
+
+If you do not already have Minikube, install it from:
+https://minikube.sigs.k8s.io/docs/start/
+
+Note: This environment has only been tested on Linux.
+
+```
+$ minikube start \
+    --mount \
+    --mount-string="/path/to/awx:/awx_devel" \
+    --cpus=4 \
+    --memory=8g \
+    --addons=ingress
+```
+
+### Verify
+
+Ensure that your AWX source code is properly mounted inside of the minikube node:
+
+```
+$ minikube ssh
+$ ls -la /awx_devel
+```
+
+## Deploy the AWX Operator
+
+Clone the [awx-operator](https://github.com/ansible/awx-operator).
+
+If you are not changing any code in the operator itself, simply run:
+
+```
+$ ansible-playbook ansible/deploy-operator.yml
```

Comment:
Might be worth mentioning that you need a few Python dependencies in addition to ansible, namely `pip install openshift`.

Reply:
Good call. Done.